changelog.d/warnings/2929.enhance.rst

```diff
@@ -0,0 +1,2 @@
+Added features to mod to be able to toggle warns being sent to both DM's and channel.
```

Comment:
Dont take it as a definitive thing but according to https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/develop/.github/CONTRIBUTING.md#47-using-towncrier

This should be `2929.feature.rst`